tests/benchmarks/fixture-bootstrap.php

```
@@ -0,0 +1,447 @@
+				throw new RuntimeException( 'Benchmark fixture setup failed: WooCommerce shipping zones are not available.' );
+			}
+
+			$zone_id = 0;
+			foreach ( WC_Shipping_Zones::get_zones() as $zone ) {
+				if ( 'Benchmark Everywhere' === ( $zone['zone_name'] ?? '' ) ) {
+					$zone_id = (int) $zone['zone_id'];
+					break;
+				}
+			}
+
+			$shipping_zone = $zone_id ? new WC_Shipping_Zone( $zone_id ) : new WC_Shipping_Zone();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '287', column '36'). (undefined)

(MissingImport)